### PR TITLE
Styling of toggle button yielded by BsDropdown via arguments is ignored 

### PR DESCRIPTION
**Pull request by zfir**
CSS not applied when using `@type` parameter on `bs-dropdown-button`

- Modify api/parameters
- Add tests

This regression has been introduce by: https://github.com/ember-bootstrap/ember-bootstrap/pull/2111/files

### CODE REVIEW

File: test-app/tests/integration/components/bs-dropdown/button-test.js [hunk 1]
```diff
@@ -16,4 +16,35 @@ module('Integration | Component | bs-dropdown/button', function (hooks) {
...
+    );
+
+    assert
+      .dom('button')
+      .hasClass('btn-primary', 'dropdown button has primary type');
+  });
+
+  test('dropdown button has correct css outline class', async function (assert) {
+    await render(
+      hbs`<BsDropdown as |dd|><dd.button @outline="true">Test</dd.button></BsDropdown>`,
```
Comment (jelhan) [inline test-app/tests/integration/components/bs-dropdown/button-test.js:32]: It should be `true` as a boolean value. Not a string.

```suggestion
      hbs`<BsDropdown as |dd|><dd.button @outline={{true}}>Test</dd.button></BsDropdown>`,
```

File: ember-bootstrap/addon/components/bs-dropdown/button.hbs [hunk 2]
```diff
@@ -4,6 +4,10 @@
   {{! template-lint-disable no-capital-arguments no-args-paths }}
   @block={{this.args.block}}
   @onClick={{@onClick}}
+  @active={{@active}}
```
Comment (jelhan) [inline ember-bootstrap/addon/components/bs-dropdown/button.hbs:7]: `@active` argument does not seem to be covered by tests. Do you mind adding tests for it?

**Review by jelhan — CHANGES_REQUESTED** [review]
Thanks a lot for fixing this bug! I likely introduced it in #2111. 😬

I have two change requests from review. Additionally Prettier complains about template syntax. Would be great if you could fix those.

File: test-app/tests/integration/components/bs-dropdown/button-test.js [hunk 3]
```diff
@@ -16,4 +16,43 @@ module('Integration | Component | bs-dropdown/button', function (hooks) {
     assert.dom('button').exists('dropdown button is a button');
     assert.dom('.dropdown-toggle').exists('has dropdown-toggle class');
   });
+
+  test('dropdown button has coorect css type class', async function (assert) {
+    await render(
+      hbs`<BsDropdown as |dd|><dd.button @type="primary">Test</dd.button></BsDropdown>`,
```
Comment (jelhan) [inline test-app/tests/integration/components/bs-dropdown/button-test.js:22]: Make Prettier happy:

```suggestion
      hbs`<BsDropdown as |dd|><dd.button @type='primary'>Test</dd.button></BsDropdown>`,
```

File: test-app/tests/integration/components/bs-dropdown/button-test.js [hunk 4]
```diff
@@ -16,4 +16,43 @@ module('Integration | Component | bs-dropdown/button', function (hooks) {
...
+      .dom('button')
+      .hasClass(
+        'btn-outline-secondary',
+        'dropdown button has secondary outline type',
+      );
+  });
+
+  test('dropdown button has correct size', async function (assert) {
+    await render(
+      hbs`<BsDropdown as |dd|><dd.button @size="lg">Test</dd.button></BsDropdown>`,
```
Comment (jelhan) [inline test-app/tests/integration/components/bs-dropdown/button-test.js:45]: Prettify the inline template:

```suggestion
      hbs`<BsDropdown as |dd|><dd.button @size='lg'>Test</dd.button></BsDropdown>`,
```

**Review by jelhan — CHANGES_REQUESTED** [review]
Two syntax changes required to make Prettier happy. I'm not sure why it enforces single code for inline templates. It should not. But that's the case for all tests currently. So nothing to be addressed here.

**Review by jelhan — APPROVED** [review]

File: test-app/tests/integration/components/bs-dropdown/button-test.js [hunk 5]
```diff
@@ -16,4 +16,43 @@ module('Integration | Component | bs-dropdown/button', function (hooks) {
     assert.dom('button').exists('dropdown button is a button');
     assert.dom('.dropdown-toggle').exists('has dropdown-toggle class');
   });
+
+  test('dropdown button has coorect css type class', async function (assert) {
```
Comment (simonihmig) [inline test-app/tests/integration/components/bs-dropdown/button-test.js:20]: very minor typo here :)

**Review by simonihmig — APPROVED** [review]
Thank you @zfir! :pray:

> CI should be fixed by https://github.com/ember-bootstrap/ember-bootstrap/pull/2130. Would prefer to land that one and fix CI before merging this. Just to make sure that we don't miss something.

Just merged that. @zfir would you mind rebasing your PR, that should then hopefully turn it fully green!

**Comment by zfir** [comment]
> Thank you @zfir! 🙏
> 
> > CI should be fixed by #2130. Would prefer to land that one and fix CI before merging this. Just to make sure that we don't miss something.
> 
> Just merged that. @zfir would you mind rebasing your PR, that should then hopefully turn it fully green!

@simonihmig just done can you check

**Comment by zfir** [comment]
@jelhan @simonihmig Can we publish a version for this? Thanks

**Comment by zfir** [comment]
Hello @jelhan @simonihmig,

Is there a version that has been published?

**Comment by jelhan** [comment]
I just released it as [v6.3.2](https://github.com/ember-bootstrap/ember-bootstrap/releases/tag/v6.3.2). I'm sorry for the delay.

**Pull request merged**